src/FlightDisplay/FlyView.qml

```diff
@@ -134,15 +125,15 @@ Item {
         color:              qgcPal.window
         visible:            false
     }
-
+    /*
```

Comment:
Can you add comments as to why this was commented out?

Reply:
removed